Use `make install` in the build script

kevinbackhouse opened the pull request:
The build script wasn't working for me, because it wasn't generating the `libc-2.XX.so`. I think the correct solution is to use the `--prefix` argument and then run `make install`.

Kyle-Kyle commented:
Looks good.
But are these two lines necessary?
~~~
mkdir -p $OUTPUT_DIR/etc
touch $OUTPUT_DIR/etc/ld.so.conf
~~~

kevinbackhouse commented:
@Kyle-Kyle: you are right, those lines are unnecessary. I added them because of this warning:

```
/localdisk/kev/how2heap/glibc_build/elf/ldconfig: Warning: ignoring configuration file that cannot be opened: /localdisk/kev/how2heap/glibc_versions/2.28/x64_tcache/etc/ld.so.conf: No such file or directory
```

But I added those lines when the build was still broken and I was trying to figure out why. It's just a harmless warning, so I'll remove those lines.

Kyle-Kyle commented:
Let's merge it then. Thanks!
